problems/1.types+operators.js

```diff
@@ -2,16 +2,26 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (val !== NaN) {
+    return true
+  } else if (val === NaN) {
+    return true
+  } else if (val === "string") {
+    return false
+  }
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+function isTruthy(val) {
+  if (val === true) {
```

Comment:
Just if (val) That will enter the if statement if truthy.. We don't care if it's literally the boolean true